src/web3_utils.py

```diff
@@ -127,26 +131,32 @@ def get_last_harvest_times(
         keeper_acl (contract): Keeper ACL web3 contract instance.
         start_block (int, optional):
             Minimum block number to start fetching harvest timestamps from. Defaults to 0.
-        etherscan_key (str)
+        chain (Network)
 
     Returns:
         dict: Dictionary of strategy addresses and their latest harvest timestamps.
     """
-    if etherscan_key is None:
-        etherscan_key = get_secret("keepers/etherscan", "ETHERSCAN_TOKEN")
+    if chain == Network.Ethereum:
+        api_key = get_secret("keepers/etherscan", "ETHERSCAN_TOKEN")
+        url = "https://api.etherscan.io/api"
+    elif chain == Network.Fantom:
+        api_key = get_secret("keepers/etherscan", "FTMSCAN_TOKEN")
```

Comment:
change the first param here to keepers/ftmscan, i'll create secret

Reply:
Done